Brickbreaker.py

```diff
@@ -0,0 +1,3 @@
+import math
+import pyaudio
+import pygame
```

Comment:
This file seems like it is not a part of your final program. You should delete any extraneous file that is not part of your program in your repository.